spec/web_helper.rb

```diff
@@ -0,0 +1,8 @@
+def sign_up (username: 'mat', email: 'mat@makers.com', password: '123abc', password_confirmation: '123abc')
```

Comment:
Metrics/LineLength: Line is too long. [107/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)<br>Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)